src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -122,7 +118,7 @@ def run_dm_dst_create(self, tool):
                 "DAOS", "/", pool1, cont6_uuid)
             cont6 = self.get_cont(pool1, cont6_uuid)
             cont6.type.update(cont1.type.value, "type")
-            self.verify_cont(cont6, False)
+            self.verify_cont(cont6, api, check_props)
```

Comment:
This should just pass False, since attrs and props aren't copied from POSIX:
```suggestion
            self.verify_cont(cont6, api, False)
```